docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+Yüksek boyutlu bir uzayda, veri manifoldundan gerçekten daha yüksek olduğundan emin olmak için enerjiyi yükseltmeniz gereken birçok bölge var. Gösterimlerin boyutları arttıkça, daha fazla negatif örneğe ihtiyaç duyulmaktadir ki enerjinin manifold üzerinde olmayan bölgelerinde daha yüksek olması şartı sağlanabilsin.  
+
+
+<!-- SimCLR shows better results than previous methods. In fact, it reaches the performance of supervised methods on ImageNet, with top-1 linear accuracy on ImageNet. The technique uses a sophisticated data augmentation method to generate similar pairs, and they train for a massive amount of time (with very, very large batch sizes) on TPUs. Dr. LeCun believes that SimCLR, to a certain extend, shows the limit of contrastive methods. There are many, many regions in a high-dimensional space where you need to push up the energy to make sure it's actually higher than on the data manifold. As you increase the dimension of the representation, you need more and more negative samples to make sure the energy is higher in those places not on the manifold. -->
+
+## [Arıtan Otokodlayıcılar](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=1384s)
+
+<!-- ## [Denoising Autoencoders](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=1384s)-->
+
+[7. haftanin uygulamasinda](https://atcold.github.io/pytorch-Deep-Learning/en/week07/07-3/), arıtan otokodlayıcıdan bahsetmiştik.
+Bu yapı, veri gösterimini bozuk girdiyi orjinal veriye geri dönüştürmeye çalışarak öğrenir. 
+Detaylıca bahsedersek, sistemi , bozulmuş veriler veri manifoldundan uzaklaştıkça karesel bir oranda büyüyen bir enerji fonksiyonu üretebilmesi için eğitiyoruz.
+
+<!-- In [week 7's practicum](https://atcold.github.io/pytorch-Deep-Learning/en/week07/07-3/), we discussed denoising autoencoder. The model tends to learn the representation of the data by reconstructing corrupted input to the original input. More specifically, we train the system to produce an energy function that grows quadratically as the corrupted data move away from the data manifold. -->
+
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-1/fig6.png" height="75%" width="75%"/><br>
+<b>Fig. 5</b>:Arıtan Otokodlayıcı Mimarisi
+</center>
+
+### Problemler
+
+<!-- ### Issues -->
+
+Ancak arıtan otokodlayıcılar ile ilgili çeşitli sorunlar bulunmaktadır. 
+Öncelikle, cok boyutlu ve sürekli bir uzayda, veriyi bozmanın sonsuz sayıda yolu vardır. Bu yüzden enerji fonksiyonunu bir sürü değişik bölgede yukarı itmenin de garantisi yokturç
+Bir diger problem ise saklı degişkenlerin noksanlığından ötürü arıtan otokodlayicilar göruntulerde kötu performans sergilemektedir.
+Bir resmi yeniden oluşturmanın sayısız yöntemi olduğundan, sistem çeşitli tahminler üretir ve bilhassa iyi öznitelikler öğrenemez. 
+Ayrıca, manifoldun ortasındak kalan bozunmuş noktalar iki taraftan da onarmamız mümkün. Bu durum enerji fonksiyonumuzda düz bölgeler oluşmasına neden olur ve genel performansı etkiler.
```

Comment:
```suggestion
Ayrıca, manifoldun ortasında kalan bozunmuş noktaları iki tarafa doğru da onarmamız mümkün. Bu durum enerji fonksiyonumuzda düz bölgeler oluşmasına neden olur ve genel performansı etkiler.
```

Reply:
I still think that taraftan is ok, but still resolving it due to other typos. Original phrase is "from both sides".